node/worker/worker.go

```diff
@@ -419,8 +430,8 @@ func New(config *params.ChainConfig, chain *core.BlockChain, engine consensus_en
 		chain:   chain,
 		engine:  engine,
 	}
-	worker.gasFloor = 500000000000000000
-	worker.gasCeil = 1000000000000000000
+	worker.gasFloor = 80000000
+	worker.gasCeil = 120000000
```

Comment:
What was meaning of old numbers , new numbers ?

Reply:
Old number is just a meaninglessly large number.